scalding-serialization-macros/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/TreeOrderedBuf.scala

```diff
@@ -122,15 +128,21 @@ object TreeOrderedBuf {
         case const: ConstantLengthCalculation[_] => (q"""
           override val staticSize: Option[Int] = Some(${const.toInt})""", q"""
           override def dynamicSize($element: $typeName): Option[Int] = staticSize""")
-        case f: FastLengthCalculation[_] => callDynamic
-        case m: MaybeLengthCalculation[_] => callDynamic
+        case f: FastLengthCalculation[_] =>
+          callDynamic
+        case m: MaybeLengthCalculation[_] =>
+          callDynamic
+        case _ => (q"""
```

Comment:
what case is this? I thought this was a sealed trait and we already had a complete match.